app/lib/actions/katello/content_view_version/incremental_update.rb

```diff
@@ -358,6 +358,18 @@ def copy_deb_content(new_repo, dep_solve, deb_ids)
         def copy_yum_content(new_repo, dep_solve, package_ids, errata_ids)
           copy_outputs = []
           if new_repo.content_type == ::Katello::Repository::YUM_TYPE
```

Comment:
optional: this method could just return early rather than having this large indented block => `return [] unless new_repo.content_type == ::Katello::Repository::YUM_TYPE`